packages/design-system/modal/modal.container.web.tsx

```diff
@@ -61,27 +57,47 @@ const ModalContainerComponent = forwardRef<ModalMethods, ModalContainerProps>(
       []
     );
 
-    useEscapeKeydown((event) => {
-      event.preventDefault();
-
-      return disableBackdropPress ? noop() : onClose?.();
-    });
-
-    // Prevent scrolling the body when the modal is open
-    useLockBodyScroll();
-
     return (
-      <FocusTrap aria-modal>
```

Comment:
did we remove the focus trap here?

Reply:
yes! see this [comment](https://github.com/showtime-xyz/showtime-frontend/pull/2150#discussion_r1182248670)

Reply:
if still exist I'll fix it again.